exec/java-exec/src/main/java/org/apache/drill/exec/ops/QueryContext.java

```diff
@@ -61,6 +61,7 @@
 // in fragment contexts
 public class QueryContext implements AutoCloseable, OptimizerRulesContext, SchemaConfigInfoProvider {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(QueryContext.class);
+  public enum StatementType {UNKNOWN, ANALYZE, CTAS, EXPLAIN, REFRESH, SELECT, SETOPTION};
```

Comment:
We don't have UNKNOWN statements. (Drill created the statement, it should know what it created.) Perhaps call this `OTHER`?

Or, better, figure out what that other statement type is and include it in the list. Throw an exception if we find we don't know the statement time because, in that case, something is wrong in the code.